website/content/api-docs/admin-partitions.mdx

```diff
@@ -175,26 +158,18 @@ deleted all the associated partitioned data in the background. Only once
 all associated data has been deleted will the Partition actually disappear.
 Until then, further reads can be performed on the partition and a `DeletedAt`
 field will now be populated with the timestamp of when the Partition was
-marked for deletion.
-
-| Method   | Path               | Produces |
-| -------- | ------------------ | -------- |
-| `DELETE` | `/partition/:name` | N/A      |
-
-This endpoint will return no data. Success or failure is indicated by the status
-code returned.
-
-The table below shows this endpoint's support for
-[blocking queries](/api/features/blocking),
-[consistency modes](/api/features/consistency),
-[agent caching](/api/features/caching), and
-[required ACLs](/api#authentication).
-
-| Blocking Queries | Consistency Modes | Agent Caching | ACL Required     |
-| ---------------- | ----------------- | ------------- | ---------------- |
-| `NO`             | `none`            | `none`        | `operator:write` |
-
-The corresponding CLI command is [`consul partition delete`](/commands/partition#delete).
+marked for deletion. This endpoint has the following characteristics:
+
+| Characteristic | Value |
+| :------------- | :---- |
+| [HTTP Method](/api-docs#http-methods) | `DELETE` |
+| [URL Path](/api-docs#version-prefix) | `/partition/:name` |
+| [Response Type](/api-docs#formatted-json-output) | none |
```

Comment:
Choose to put "none" rather than "N/A" for the value here because, technically, a DELETE request _can_ respond with what was deleted, we just choose not to in Consul.